taxonomy-manager-common/src/main/java/com/digirati/taxman/common/rdf/identity/IdResolver.java

```diff
@@ -11,6 +12,17 @@
     private IdResolver() {
     }
 
+    public static Optional<UUID> resolve(String uri, Pattern pattern) {
+        if (uri == null)
```

Comment:
Braces.

Reply:
Also, `uri` should never be null as far as I can tell.

Reply:
It can be during the `create` calls.

Good call on braces. I might look into setting that as an error in InteliJ